blog/2023-10-29-ratify-1.0.0.mdx

```diff
@@ -0,0 +1,93 @@
+It helps users to validate the image integrity and authenticity in containers secure supply chain, ensures your deployments are built from a trusted entity and that images haven't been tampered with since their creation.
+![](/img/ratify-e2e.png)
+
+Ratify was [launched and open-sourced](https://cloudblogs.microsoft.com/opensource/2021/12/09/ratify-container-supply-chain-in-kubernetes/) on December 9, 2021. After the fast iteration with 18 releases over the past two years, the core framework becomes mature and extensible now and reaches enterprise-grade stability. Users can easily run Ratify as an External Data Provider for Gatekeeper admission controller on Kubernetes with Helm.
+
+## Notable capabilities in this Release
+
+- Ratify as a CLI binary for verifying artifacts stored in a registry
+- Support multiple verifiers including Notation, Cosign, and SBOM
+- Support dynamic configuration through CRD
+- TLS certificate management and rotation for mTLS service-to-service communication
+- Out-of-the-box observability: logging and prometheus metrics support
+- Extensible plugin design to support new verifier and referrer store plugins
+- High availability support (Experimental) 
+    
+## Extensible plugin design
+
+Ratify framework is designed from the beginning on an extensible plugin model without any affinity towards a particular verifier or storage model. 
+Ratify uses extensible plugin architecture to enable extensibility and interoperability to support both the existing and new emerging needs of the artifact verification. Developers can easily [create a new plugin](https://ratify.dev/docs/1.0/quickstarts/creating-plugins) and integrate into Ratify ecosystem. See [Framework overview](https://ratify.dev/docs/1.0/reference/ratify-framework-overview) for details. 
+
+![](/img/ratify-core-principals.png)
+
+## Support dynamic configuration through CRDs
+
+Ratify supports updating its common configuration through Kubernetes CRDs using kubectl commands, delivers a Kubernetes-native experience for users. The configuration takes effect dynamically so users no need to restart the running Ratify. There are four CRDs including Certificate Store, Policy, Store, and Verifier. You can see [CRD reference docs](https://ratify.dev/docs/1.0/reference/crds/certificate-stores) for details.
+
+![](/img/ratify-crd.png)
+
+## Observability
+
+Observability is the process of gaining insight into the behavior and performance of applications running on Kubernetes, in order to monitor, troubleshoot, and identify issues more effectively. 
+Ratify introduces standardized logging component and metrics exporter for Prometheus in this release so that users can gather and visualize the metrics via the popular monitoring and logging stack, like Prometheus, Grafana, and EFK.
```

Comment:
Reword: "...the metrics via the popular ..." to "...the metrics via a popular monitoring and logging stack with Prometheus, Grafana, and EFK."